docs/docs/develop/import-data.md

```diff
@@ -33,7 +33,13 @@ path: /path/to/data.csv
 
 Rill will ingest the data next time you run `rill start`.
 
-Note that if you provide a relative path, the path should be relative to your Rill project root (where your `rill.yaml` file is located), not relative to the `sources` directory.
+Note that if you provide a relative path, *the path should be relative to your Rill project root* (where your `rill.yaml` file is located), **not** relative to the `sources` directory.
```

Comment:
`*the path should be relative to your Rill project root*` This shows up as italic right now, shouldn't this be bold?

Reply:
I had intentionally italicized this part, since I bolded a part later, to avoid overusing the bolding. Open to formatting changes though.